app/code/Magento/CatalogWidget/Model/Rule/Condition/Product.php

```diff
@@ -38,6 +38,13 @@ class Product extends \Magento\Rule\Model\Condition\Product\AbstractProduct
      */
     protected $storeManager;
 
+    /**
+     * Attribute data key that indicates whether it should be used for rules
+     *
+     * @var string
+     */
+    protected $_isUsedForRuleProperty = 'is_used_for_promo_rules';
```

Comment:
This is a static value. You should use a constant for this. For a start it is more efficient.